verify/verifier.go

```diff
@@ -183,11 +191,26 @@ func (v *Verifier) verifySecurityParams(blobHeader BlobHeader, batchHeader *disp
 	}
 
 	// ensure that required quorums are present in the confirmed ones
-	for _, quorum := range v.cv.quorumsRequired {
+	for _, quorum := range requiredQuorum(int64(batchHeader.ReferenceBlockNumber), v) {
 		if !confirmedQuorums[quorum] {
 			return fmt.Errorf("quorum %d is required but not present in confirmed quorums", quorum)
 		}
 	}
 
 	return nil
 }
+
+func requiredQuorum(referenceBlockNumber int64, v *Verifier) []uint8 {
+	// This check is required due to a bug we had when we updated the EigenDAServiceManager in Holesky. For a brief period of time, the quorum 1 was not
+	// required for the commitment to be confirmed, which will not validate archive blobs required for full syncs (archive nodes).
+	// This check is only for testnet and for a specific block range.
```

Comment:
```suggestion
	// required for the commitment to be confirmed, so the disperser created batches with only quorum 0 signatures. 
	// Archive nodes trying to sync from these stored batches would thus fail validation here since
	// quorumsRequired is read from the latestBlock, where the bug has been fixed and both quorums are required.
	// This check is only for testnet and for a specific block range.
```